index.rst

```diff
@@ -6,24 +6,474 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
+.. Make in-text citations with: :cite:`bibkey`.
+.. Uncomment to use citations
+.. .. rubric:: References
+.. 
+.. .. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
+..    :style: lsst_aa
 
 Abstract
 ========
 
-Principles, guidelines, conventions, formatting style, and workflow for tutorials developed by the Community Engagement Team (CET).
+A guideline for anyone creating or updating tutorials for the Rubin science community.
+This includes all community-facing demonstrations of how to use the Rubin Science Platform or how to analyze LSST data.
+
+Pedagogical principles
+======================
+
+All tutorial developers should adhere to the following principles.
+
+Inclusivity
+-----------
+
+Avoid offensive or exclusionary language at all times (e.g., violent or ableist terms).
+Ensure jargon and acronyms are defined.
+
+Level-appropriate
+-----------------
+
+Tutorials should clearly identify and teach to their target audience (beginner, intermediate, or advanced).
+
+Skill-focused
+-------------
+
+Tutorials should focus on teaching one or a few new skills or techniques, or providing one scientific demonstration.
+
+Consumable
+----------
+
+The user should be able to work through the core contents of a tutorial within 30 minutes.
+Additional examples could be included, but should be clearly labeled as extra steps.
+
+Well-documented
+---------------
+
+High quality documentation should be provided with the tutorial, including narrative descriptions, citations, references, 
+and external links to, e.g., package documentation.
+
+Cross-referenced
+----------------
+
+Tutorials should reference an precursor or advanced tutorials that users should consider as pre-requisit or follow-up resources.
```

Comment:
Extra "an" in "...an precursor..."?

Reply:
done, also corrected "pre-requisit" to "prerequisite"